tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           docker run --rm --volume /data/onnx:/data/onnx:ro --volume $(Build.SourcesDirectory):/onnxruntime_src --volume $(Build.BinariesDirectory):/build --volume /data/models:/build/models:ro \
           --volume $HOME/.onnx:/home/onnxruntimedev/.onnx -e NIGHTLY_BUILD onnxruntimeregistry.azurecr.io/internal/azureml/onnxruntimecpubuild:ch2j /opt/python/cp37-cp37m/bin/python3 \
           /onnxruntime_src/tools/ci_build/build.py --build_dir /build --config Release --enable_lto \
-          --skip_submodule_sync  --parallel --build_shared_lib --use_openmp
+          --skip_submodule_sync  --parallel --build_shared_lib --disable_ort_format_load --use_openmp
```

Comment:
If we're disabling it in one pipeline, I suggest doing it in all pipelines that generate public pkgs so that the binary produced in different packages is generated from the same set of build flags. Having the DLL in the zip pkgs (produced by this pipeline) different from the one in the nuget can cause confusion; we'd one such case before. If the savings are not huge (only 60kb), we can just keep the status quo.

Reply:
I agree 60kb saving is not worth the confusion, maybe in a future release we can publish a standalone conversion tool depends on how popular the mobile format will go, and disable the save/load flatbuffer ort format completely in the full ORT (only enable it in minimal ORT)

Reply:
Please update the nocontrib ops pipelines as well as they're used by vscode and Edge. If you search for --enable_msvc_static_runtime, you'll know the places where an update is required.